src/plugins/vis_type_pie/public/pie_fn.ts

```diff
@@ -133,7 +134,33 @@ export const createPieVisFn = (): VisTypePieExpressionFunctionDefinition => ({
     } as PieVisParams;
 
     if (handlers?.inspectorAdapters?.tables) {
-      handlers.inspectorAdapters.tables.logDatatable('default', context);
+      const logTable = prepareLogTable(context, [
+        [
+          [args.metric],
+          i18n.translate('visTypePie.function.args.labelsHelpText', {
+            defaultMessage: 'Slice size',
+          }),
+        ],
+        [
+          args.buckets,
+          i18n.translate('visTypePie.function.args.labelsHelpText', {
+            defaultMessage: 'Slice',
+          }),
+        ],
+        [
+          args.splitColumn,
+          i18n.translate('visTypePie.function.args.labelsHelpText', {
+            defaultMessage: 'Column Split',
```

Comment:
I think that in order to be in sync with our writing guidelines the text should change to `Column split` and `Row split`